client/client-multi/client-java-core/src/main/java/org/kaaproject/kaa/client/Kaa.java

```diff
@@ -43,14 +42,30 @@ public static KaaClient newClient(KaaClientPlatformContext context) throws KaaRu
     }
 
     public static KaaClient newClient(KaaClientPlatformContext context, KaaClientStateListener listener) throws KaaRuntimeException {
+        return newClient(context, listener, false);
+    }
+
+    /**
+     * Creating new Kaa client
+     *
+     * @param context             - Kaa platform specific context for client init
+     * @param listener            - notifier for changing client state
+     * @param isAutogeneratedKeys - check if creating public/private key pair.
```

Comment:
isAutogeneratedKeys - whether to generate public/private key pair automatically.